griptape/tasks/text_to_image_task.py

```diff
@@ -25,6 +25,10 @@ class TextToImageTask(BaseImageGenerationTask):
         output_file: If provided, the generated image will be written to disk as output_file.
     """
 
+    DEFAULT_INPUT_TEMPLATE = "{{ args[0] }}"
+
+    _input: str | TextArtifact | Callable[[BaseTask], TextArtifact] = field(default=DEFAULT_INPUT_TEMPLATE)
```

Comment:
There's some duplication between TextToImageTask and BaseTextInputTask to satisfy Pyright's clean inheritance standards. Could inputs be refactored to mixins to prevent this duplication?

Reply:
In the past, yes, but this no longer inherits from BaseTextInputTask so DEFAULT_INPUT_TEMPLATE doesn't exist in any of its parents/ancestors.